### PR TITLE
added demo on gh-pages

### PR DESCRIPTION
**Pull request by BeastJsWeb**
Lets's check. On my repo on GitHub, all works fine.
https://beastjsweb.github.io/SimpleCart-2.0/

### CODE REVIEW

**Review by molotow11 — CHANGES_REQUESTED** [review]
Looks awesome!
Please fix the following:
- Russian translations in the table header
- P into $ or add currency parameter if possible
- Fix (x) close icons from (x) letter into ascii char

**Comment by BeastJsWeb** [comment]
Nice. Sure

**Review by molotow11 — APPROVED** [review]

File: demo/index.html [hunk 1]
```diff
@@ -46,7 +46,7 @@ <h2>Must be uploaded to the site for make cookies work (not working as local fil
 					<td role="cell">90</td>
 					<td role="cell">5400</td>
 					<td role="cell">176</td>
-					<td class="price" role="cell">2087.79 Р</td>
+					<td class="price" role="cell">208.79 $</td>
```
Comment (molotow11) [inline demo/index.html:49]: @BeastJsWeb Please try to think how it is possible to dynamically fix currency here?

File: demo/simple-cart-demo.js [hunk 2]
```diff
@@ -100,8 +101,13 @@ document.addEventListener("DOMContentLoaded", () => {
...
-          <span class='${createClass(productSelectors.title)}'>${product.title}</span>
-          <span class='${createClass(productSelectors.price)}'>${product.price} Р</span>
+          <span class='${createClass(productSelectors.title)}'>${
+            product.title
+          }</span>
+          <span class='${createClass(productSelectors.price)}'>${
+            product.price} ${
+            productSelectors.currency
+          }</span>
           <span class='qty-sign'>x</span>
```
Comment (molotow11) [inline demo/simple-cart-demo.js:111]: @BeastJsWeb Please fix (x) here as well

File: simple-cart.js [hunk 3]
```diff
@@ -100,8 +101,13 @@ document.addEventListener("DOMContentLoaded", () => {
...
-          <span class='${createClass(productSelectors.title)}'>${product.title}</span>
-          <span class='${createClass(productSelectors.price)}'>${product.price} Р</span>
+          <span class='${createClass(productSelectors.title)}'>${
+            product.title
+          }</span>
+          <span class='${createClass(productSelectors.price)}'>${
+            product.price} ${
+            productSelectors.currency
+          }</span>
           <span class='qty-sign'>x</span>
```
Comment (molotow11) [inline simple-cart.js:111]: And here

**Review by molotow11 — COMMENTED** [review]
Great!

File: demo/index.html (hunk 1 above)
Comment (BeastJsWeb) [reply demo/index.html:49]: Ok, I will see

File: demo/simple-cart-demo.js (hunk 2 above)
Comment (BeastJsWeb) [reply demo/simple-cart-demo.js:111]: Ok

File: simple-cart.js (hunk 3 above)
Comment (BeastJsWeb) [reply simple-cart.js:111]: Ok

**Pull request merged**